tests/api.test.ts

```diff
@@ -1,46 +1,45 @@
-import { describe, afterEach, beforeEach, it, test, expect, vi } from 'vitest';
-import { APIRequest, APIRequest2, defaultRequestUri, request } from './api';
-import crossFetch from 'cross-fetch';
+import { describe, beforeEach, it, test, expect, vi, afterEach } from 'vitest';
+import { APIRequest, APIRequest2, defaultRequestUri, request } from './api.js';
+import { createFetchMock, type FetchMock } from '../src/index.js';
+
+describe('testing mockResponse', () => {
+  const fetch = createFetchMock({ autoEnable: true });
 
-describe('testing mockResponse and alias once', () => {
-  beforeEach(() => {
-    fetch.resetMocks();
-  });
   it('mocks a response', async () => {
-    fetch.mockResponseOnce(JSON.stringify({ secret_data: 'abcde' }, { status: 200 }));
+    fetch.mockResponseOnce(JSON.stringify({ secret_data: 'abcde' }), { status: 200 });
```

Comment:
Wow, was this just a bug in the tests previously?

Reply:
Yes!